modules/tinymce/CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Aligning a table to the left or right will now use margin styling instead of float styling #TINY-6558
 - The `tinymce.settings` global property is no longer set upon initialization #TINY-7359
 - Addons such as plugins and themes are no longer constructed using the `new` operator #TINY-8256
+- A number of APIs that were not actually classes no longer require using the `new` operator #TINY-8322
```

Comment:
```suggestion
- A number of APIs that were not classes no longer require using the `new` operator #TINY-8322
```
I suspect Tyler won't like using "actually". Could use "proper" maybe, if you want something there?

Reply:
We need something there to show they were "fake"/"partial" classes and I'm not sure `proper` covers that. It's better than nothing though but I'm going to ask @tylerkelly13 to weigh in.

Reply:
```suggestion
- A number of APIs that were not proper classes, are no longer constructed using the new operator #TINY-8322
```
Was talking to Tyler via DM's and this is the best we've come up with so far :shrug: